_topics/en/transaction-pinning.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Transaction pinning
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Security Problems
+  - Fee Management
+  - Contract Protocols
+  - Transaction Relay Policy
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Transaction pinning** is a method for making fee bumping
+  prohibitively expensive by abusing node protections against
+  attacks that can waste bandwidth, CPU, and memory.  This can make fee
+  management more difficult in multipart contract protocols (such as LN).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Nodes such as Bitcoin Core that allow transactions to be replaced
+  (RBF) or packaged with higher-fee child transactions (CPFP) place
+  restrictions on those replacements in order to prevent various DoS
+  attacks.  However, when two or more people each have the ability to
+  fee bump a transaction, this makes it possible for one of them to
+  pin their version of a transaction at one of the limits and prevent
```

Comment:
Since you're introducing the term _pin_ for the first time here, perhaps use _italics_ or **bold**

Reply:
done